docs/content/en/docs/tutorials/tetragon-metrics.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Tetragon Metrics"
+weight: 1
+icon: "overview"
+description: "Fetching and understanding Tetragon metrics"
+---
+
+# Fetching Tetragon Metrics
+
+## Kubernetes
+
+Tetragon, when installed with Helm as mentioned in [Deploying on Kubernetes] ({{< ref "docs/getting-started/deployment/kubernetes" >}}), creates a pod with `tetragon-` prefix. It also creates a service named `tetragon` that exposes metrics on the specified port.
+
+### Getting metrics port
+
+Check if `tetragon` service is running
+
+```bash
+➜  ~ kubectl get services -n kube-system
+NAME       TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)                  AGE
+kube-dns   ClusterIP   10.96.0.10     <none>        53/UDP,53/TCP,9153/TCP   3m
+tetragon   ClusterIP   10.96.54.218   <none>        2112/TCP                 3m
+```
+
+Describe the tetragon service to see the port on which metrics are being exposed.
+
+```bash
+➜  ~ kubectl describe service -n kube-system tetragon
+Name:              tetragon
+Namespace:         kube-system
+Labels:            app.kubernetes.io/instance=tetragon
+                   app.kubernetes.io/managed-by=Helm
+                   app.kubernetes.io/name=tetragon
+                   helm.sh/chart=tetragon-0.11.0
+Annotations:       meta.helm.sh/release-name: tetragon
+                   meta.helm.sh/release-namespace: kube-system
+Selector:          app.kubernetes.io/instance=tetragon,app.kubernetes.io/managed-by=Helm,app.kubernetes.io/name=tetragon,helm.sh/chart=tetragon-0.11.0
+Type:              ClusterIP
+IP Family Policy:  SingleStack
+IP Families:       IPv4
+IP:                10.96.54.218
+IPs:               10.96.54.218
+# Metrics Port is 2112
+Port:              metrics  2112/TCP
+TargetPort:        2112/TCP
+Endpoints:         172.18.0.2:2112
+Session Affinity:  None
+Events:            <none>
+```
+
```

Comment:
```suggestion
Note on the previous output that 2112 is the port on which the service is listening. 
It is also the port on which Tetragon listens for its metrics server with the default
Helm values.
```
I think you can remove that part since we already see the port in the previous command output.

Reply:
Sure, I added a separate note to it.